lib/Conversion/TorchToLinalg/TensorConstructors.cpp

```diff
@@ -79,6 +81,227 @@ class ConvertAtenConstantPadNdOp
 };
 } // namespace
 
+namespace {
+
+  // Lower aten.replication_pad2d operator into a sequence of
+  // tensor.extract_slice and tensor_insert_slice operations.
```

Comment:
It looks like you're lowering into `aten.cat` ops rather than `insert_slice`. Does this comment need updating?

Reply:
Thanks, suggested changes have been made.